spec/features/merchants/invoice/show_spec.rb

```diff
@@ -50,5 +50,18 @@
 
       expect(page).to have_content("Total Revenue: 1000")
     end
+#   18. Merchant Invoice Show Page: Update Item Status
+    it "can update an invoice_item's status with a select field" do 
+      visit "/merchants/#{@merchant1.id}/invoices/#{@invoice1.id}"
+
+      expect(page).to have_select("itemstatus")
+      expect(page).to have_field("itemstatus", with: "pending")
+      
+      select "shipped", from: "itemstatus"
```

Comment:
select forms are cool, had no idea how these worked until you completed this one so thank you!